### DISCUSSION TITLE
Problems importing packages

### DISCUSSION THREAD

**Discussion by xxtemps** [ansys/pyansys-geometry#1136] [Q&A] [answered]
![1713426245032](https://github.com/ansys/pyansys-geometry/assets/12379914/79086ecf-8723-4cde-a796-c3000b003862)

Hello, why can't I download the package

**Comment by RobPasMue** [comment, marked as answer]
Hi @xxtemps 

The PyAnsys Geometry package is installed as follows:

```bash
pip install ansys-geometry-core
```

Once your package is installed, if you open a Python session you can import the different subpackages and modules as follows:

```py

from ansys.geometry.core.shapes import ...

```

Hope this helps!

**Comment by xxtemps** [comment]
![image](https://github.com/ansys/pyansys-geometry/assets/12379914/15c13e2a-f38a-4ac8-865a-7b0531866fd6)
![image](https://github.com/ansys/pyansys-geometry/assets/12379914/a61ddef3-3d27-4219-9e71-42101262e3ec)

it's still get an error

**Reply by RobPasMue** [reply]
Hi @xxtemps - could you please run the following command on your terminal?

```
python -c "from ansys.geometry.core import __version__; print(__version__)"
```

**Reply by RobPasMue** [reply]
Let me know the output

**Comment by xxtemps** [comment]
0.4.11

**Reply by RobPasMue** [reply]
That's the main problem. The ``shapes`` subpackage is only available starting on version ``0.5.0``. You can force the upgrade as follows:

```
pip install --force-reinstall -v "ansys-geometry-core==0.5.0"
```

**Comment by xxtemps** [comment]
Okay, thank you.

**Reply by RobPasMue** [reply]
Anytime 😄
